src/Raven.Studio/typescript/viewmodels/database/settings/timeSeries.ts

```diff
@@ -141,42 +141,55 @@ class timeSeries extends viewModelBase {
     private initTooltips() {
         popoverUtils.longWithHover($(".aggregation-info"),
             {
-                content: `<ol style="max-width: 400px;">
-                                    <li>
-                                        <small>
-                                            <strong>New time-series data</strong> is created for each aggregation policy that is defined.
-                                            <br />
-                                            Its name will be &lt;original-time-series-name&gt;_&lt;additional-policy-name&gt;
-                                        </small>
-                                    </li>
-                                    <li>
-                                        <small>
-                                            The time-series data generated by the <strong>first</strong> aggregation policy is based on the Raw time-series data.
-                                            After that, the input data for each a aggregation policy will be the time-series data that was generated by the <strong>previous</strong> policy. <br />
-                                            The first Aggregation Rule is based on the Time Series data.<br />
-                                            Each of the following Aggregation Rules is based on data from its <strong>preceding</strong> rule.
-                                        </small>
-                                    </li>
-                                </ol>`,
+                content: `
+                         <small><br>
+                             <ul>
+                                 <li>
+                                     <strong>A new time series</strong> is created by each Aggregation Policy defined.<br>
+                                     Its name will be <code>&lt;Raw-data-time-series-name&gt;@&lt;Aggregation-policy-name&gt;</code>
+                                  </li><br>
+                                  <li>
+                                     Each entry in such auto-generated time series has the following values:<br>
+                                     <i>First, Last, Min, Max, Sum, Count</i><br>
+                                     which is rolled-up data <strong>based only on the Aggregation Time Frame defined by the policy!</strong>
```

Comment:
I wonder is it worth to mention what happens when we have 2 values in input collection?

Reply:
Added
![image](https://user-images.githubusercontent.com/17376974/86528912-3a9dce80-beb5-11ea-9d42-924d7f18f1f0.png)